index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang='en'>
+  <head>
+    <meta charset='UTF-8'/>
+    <meta name="description"/>
+    <title>Sedona cosik</title>
+  </head>
+
+  <body>
+    <div class="wrapper"><!-- main page -->
+      <header class="page-header">
+        <div calss="page-header__backgroun-img">
+          <nav class="main-nav"><!-- nav menu-->
+            <ul class="main-nav__list">
+              <li class="main-nav__item main-nav__item--active"><a href="index.html">Home</a></li>
+              <li class="main-nav__item"><a href="photos.html">Photos and videos</a></li>
+              <li class="main-nav__item"><a href="form.html">Feedback Form</a></li>
+              <li class="main-nav__item">HTML Academy</li>
+            </ul>
+            <button href="" class="main-nav__toggle main-nav__toggle--close">
+              <img src="" alt="Menu button" class="main-nav__button">
+            </button>
+            <img src="" class="main-nav__logo" alt="Sedona logo"> <!-- LOGO -->
+          </nav><!-- nav menu ends-->
+
+          <ul class="intro-img">
+            <li><img src="" class="intro-img--top" src="/" alt="Welcome text"></li>
```

Comment:
To be hones I don't know which one is better to use. but I assume that it  will be easier to hide text..
I had a quick search for the answer and i think it will be better to fallow with your example.